### DISCUSSION TITLE
Bypass Login Screen not working

### DISCUSSION THREAD

**Discussion by Steffeng5** [requarks/wiki#4099] [Error / Bug Report] [answered]
Beside the local auth provider I setup Keycloak and activated both options in security settings: Hide Local Provider and Bypass Login Screen. But only the hiding the local provider is working. By navigation to /login I see the list with one Item (Keycloak) I have to click on to login. 

Is this maybe a bug? I cannot find anything regarding this.

(I'm using latest docker image based on 2.5.201 )

**Comment by BerMan7328** [comment]
Same problem here, Did You found some solution?
version 2.5.285

**Comment by joeldeteves** [comment]
I'm on latest (2.5.289) and I have the same problem, using Azure AD as my authenticator

**Comment by joeldeteves** [comment, marked as answer]
Hey guys, @ngpixel answered this for me in the Slack channel: You need to re-order the Authentication methods so your preferred method is on the top of the list.

Once I dragged Azure to the top, the bypass redirect started working.

Thanks for the quick answer and solution, @ngpixel

**Reply by PythonCoderAS** [reply]
I wasn't able to move the authentication methods manually and had to manually edit the database to reorder them. Is that intended?

**Reply by joeldeteves** [reply]
I haven't run into this issue - for me it was a matter of using the drag and drop functionality in the UI. Did you try another browser? Almost any flavor of Chrome / Chrome derivatives should work. Just click on the authentication methods and drag them around to reorder them

**Reply by PythonCoderAS** [reply]
Oh I see the icons on the left of each method is for reordering. I thought that was the icon for the authentication method, my bad.

![Screenshot 2023-10-24 at 8 55 24 PM](https://github.com/requarks/wiki/assets/13932583/328d20d9-04f1-44c7-b61a-3d1a66d16914)
